File: backend/patch_test.py
import re

with open('backend/tests/Feature/Api/V2/CurrentProfileControllerTest.php', 'r') as f:
    content = f.read()

content = content.replace('$response->assertCreated();', '$response->dd();\n        $response->assertCreated();')
content = content.replace('$response->assertOk();', '$response->dd();\n        $response->assertOk();')

with open('backend/tests/Feature/Api/V2/CurrentProfileControllerTest.php', 'w') as f:
    f.write(content)

File: backend/patch_test.php
<?php
$content = file_get_contents('tests/Feature/Api/V2/ReportCardControllerTest.php');
$content = str_replace("->dump()->assertOk()", "->assertOk()", $content);
$content = str_replace("->dump()->assertStatus(403)", "->assertStatus(403)", $content);
$content = str_replace("clone \$student->id", "\$student->id", $content);
file_put_contents('tests/Feature/Api/V2/ReportCardControllerTest.php', $content);
